templates/docs/index.rst.jj2

```diff
@@ -0,0 +1,13 @@
+{% macro underlines(name) %}
```

Comment:
I think it might be possible to share this macro with `templates/docs/source/index.rst.jj2`